### PR TITLE
feat(LH-73627): Generate Postman Collection

### PR DESCRIPTION
**Pull request by weilueluo**
### Description
- added generate postman collection as part of the cli script
- generated a `postman-collection.json` as a one time thing

### CODE REVIEW

File: object-service-openapi.yaml
Comment (weilueluo) [inline object-service-openapi.yaml:1]: This file is just ordering changes

File: package-lock.json [hunk 1]
```diff
@@ -45,6 +46,11 @@
       "resolved": "https://registry.npmjs.org/@exodus/schemasafe/-/schemasafe-1.3.0.tgz",
       "integrity": "sha512-5Aap/GaRupgNx/feGBwLLTVv8OQFfv3pq2lPRzPg9R+IOBnDgghTGW7l7EuVXOvg5cc/xSAlRW8rBrjIC3Nvqw=="
     },
+    "node_modules/@faker-js/faker": {
```
Comment (hazital) [inline package-lock.json:49]: just confirming - all the new imports are necessary, right?

**Review by hazital — APPROVED** [review]
Really nice! just wondering how Devnet will pick the `postman-collection.json` file. don't we need to update the config.json file, or does the devnet just know to pick that file by its name?

File: package-lock.json (hunk 1 above)
Comment (weilueluo) [reply package-lock.json:49]: Yea I ran a `npm ci` before commit 🙂

**Comment by weilueluo** [comment]
> Really nice! just wondering how Devnet will pick the `postman-collection.json` file. don't we need to update the config.json file, or does the devnet just know to pick that file by its name?

It is something I need to configure in pubhub, and I will configure it such that it picks up the file of this name

**Pull request merged**